README.md

```diff
@@ -71,6 +72,7 @@ Choose what you find more suitable for you:
 - [Extreme Rate Shells](https://extremerate.com/collections/steam-deck-shells) - Extreme Rate Steam Deck Shells.
 - [GamingCobra Shells](https://www.gamingcobra.com/collections/steam-deck-shells) - Gaming Cobra Steam Deck Shells.
 - [JSAUX Transparent Back Plate](https://jsaux.com/products/transparent-back-plate-for-steam-deck-pc0106) - JSAUX Transparent Back Plate for Steam Deck.
+- [Best Xda Accessories](https://www.xda-developers.com/best-steam-deck-accessories/) - Best Steam Deck accessories by XDA.
```

Comment:
What do you think about exploding this link with the list inside of it?  
I can help you with this if you need.

Reply:
Don't worry, I wasn't sure about the list anyway, I was waiting for your feedback.!

Reply:
Updated